contracts/SSVNetworkViews.sol

```diff
@@ -70,16 +70,16 @@ contract SSVNetworkViews is
 
     function getOperatorById(
         uint64 operatorId
-    ) external view override returns (address, uint256, uint32) {
+    ) external view override returns (address, uint256, uint32, bool) {
         (
             address operatorOwner,
             uint64 fee,
             uint32 validatorCount,
-
+            Snapshot memory snapshot
         ) = _ssvNetwork.operators(operatorId);
-        if (operatorOwner == address(0)) revert OperatorDoesNotExist();
+        if (snapshot.block == 0) revert OperatorDoesNotExist();
```

Comment:
I dont think this is good, because the operator "does" exist i think we should return with all 0 except operatorOwner please double check with Ariel

Reply:
Updated to return owner address, empty amounts and false as active if the operator was removed.